packages/velaux-ui/src/components/WorkflowStudio/type-select.tsx

```diff
@@ -24,44 +24,22 @@ interface DefinitionCatalog {
   definitions: DefinitionBase[];
 }
 
-const defaultDefinitionCatalog: Record<string, string> = {
-  'step-group': 'Group',
-  deploy: 'Delivery',
-  'apply-app': 'Delivery',
-  'apply-deployment': 'Delivery',
-  'apply-component': 'Delivery',
-  'addon-operation': 'Delivery',
-  'deploy-cloud-resource': 'Delivery',
-  'share-cloud-resource': 'Delivery',
-  'export-service': 'Delivery',
-  notification: 'Notification',
-  webhook: 'Notification',
-  'create-config': 'Config Management',
-  'delete-config': 'Config Management',
-  'list-config': 'Config Management',
-  'read-config': 'Config Management',
-  'export-data': 'Config Management',
-  export2config: 'Config Management',
-  export2secret: 'Config Management',
-  suspend: 'Notification',
-};
-
 const defaultCatalog: Record<string, DefinitionCatalog> = {
-  Group: {
```

Comment:
This category is very important, why do we delete it? The `step-group` step type should belong to this category. 

BTW, We can also turn the `step-group` into a stationary button as another solution.

Reply:
I deleted it because it falls under `Process Control` category , as mentioned [here](https://github.com/kubevela/kubevela/blob/4a16b2f6fef159cbbc02e4cd627f8bb3ec37d4e8/charts/vela-core/templates/defwithtemplate/step-group.yaml#L7).
I should have prioritised it while sorting.

Reply:
> BTW, We can also turn the `step-group` into a stationary button as another solution.
I think this needs some discussion about where to put the button.

Reply:
Oh, let's make `Process Control` the first category.

Reply:
Sure, I'm on it.